onnxruntime/test/common/random_generator_test.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "gtest/gtest.h"
+#include "test/common/tensor_op_test_utils.h"
+#include <vector>
+
+namespace onnxruntime {
+namespace test {
+
+TEST(TensorGenerator, DiscreteFloat) {
+  FixedPatternValueGenerator random{};
```

Comment:
nit: `random` seems to be a less fitting name now :)

Reply:
It's merged but will fix if I touch the code again.